lookout/style/format/benchmarks/release_v0.1/30-seconds-of-code

```diff
@@ -0,0 +1,82 @@
+[1;36mINFO[0m:94f5:QualityReportAnalyzer:[0mClassification report:
+                   precision    recall  f1-score   support
```

Comment:
Missing header: analyzer name, SDK version, package version, Git hash, date and time, Python runtime version, Python package versions on which we depend on (`pipdeptree --packages lookout-style`, [pipdeptree](https://pypi.org/project/pipdeptree/), should be put in `<details></details>`)

Reply:
I will report it for the summary report to avoid duplication of these data. Still needs to be done.